ProjFS.Linux/PrjFSLib.Linux.Managed/VirtualizationInstance.cs

```diff
@@ -186,6 +206,117 @@ public virtual void StopVirtualizationInstance()
             throw new NotImplementedException();
         }
 
+        private static string GetProcCmdline(int pid)
+        {
+            using (var stream = System.IO.File.OpenText(string.Format("/proc/{0}/cmdline", pid)))
+            {
+                string[] parts = stream.ReadToEnd().Split('\0');
+                return parts.Length > 0 ? parts[0] : string.Empty;
```

Comment:
Oooh, that's nice too!

Reply:
Turns out I actually miss working on C# 😅  (It didn't help that last time I was using it for work, it was -- in part -- on services integrating with SharePoint. Y I K E S.)